tests/backend/test_design_step_bbox.py

```
@@ -0,0 +1,99 @@
+from pathlib import Path
+from typing import Dict, Tuple
+
+import httpx
+import pytest
+from fastapi.testclient import TestClient
+
+from api.config import cache_clear
+from api.main import app
+
+cadquery = pytest.importorskip("cadquery", reason="CadQuery is required to parse STEP files")
+from cadquery.occ_impl import importers as cq_importers  # noqa: E402  # pylint: disable=wrong-import-position
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)